asciidoc-confluence-publisher-converter/src/test/java/org/sahli/asciidoc/confluence/publisher/converter/AsciidocConfluencePageTest.java

```diff
@@ -1062,6 +1062,42 @@ public void renderConfluencePage_asciiDocWithInlineImage_returnsConfluencePageWi
         assertThat(asciidocConfluencePage.content(), is(expectedContent));
     }
 
+    @Test
+    public void renderConfluencePage_asciiDocWithInlineRemoteImage_returnsConfluencePageWithInlineRemoteImage() {
+        // arrange
+        String adocContent = "Some text image:https://asciidoctor.org/images/octocat.jpg[GitHub mascot]";
+        AsciidocPage asciidocPage = asciidocPage(prependTitle(adocContent));
+
+        // act
+        AsciidocConfluencePage asciidocConfluencePage = newAsciidocConfluencePage(asciidocPage, UTF_8, TEMPLATES_FOLDER, dummyAssetsTargetPath());
+
+        // assert
+        String expectedContent = "<p>Some text " +
+                "<ac:image ac:alt=\"GitHub mascot\">" +
+                "<ri:url ri:value=\"https://asciidoctor.org/images/octocat.jpg\"></ri:url>" +
+                "</ac:image></p>";
+        assertThat(asciidocConfluencePage.content(), is(expectedContent));
+        assertThat(asciidocConfluencePage.attachments().size(), is(0));
+    }
+
+    @Test
+    public void renderConfluencePage_asciiDocWithBlockRemoteImage_returnsConfluencePageWithBlockRemoteImage() {
```

Comment:
The test also uses a block title as image caption, so maybe `renderConfluencePage_asciiDocWithBlockRemoteImageAndBlockTitle_returnsConfluencePageWithBlockRemoteImageAndImageCaption` ?